test/functional/plugin/lsp_spec.lua

```diff
@@ -3597,6 +3597,94 @@ describe('LSP', function()
       eq(expected, qflist)
     end)
 
+    it('Opens the loclist if config contains `loclist`', function()
+      clear()
```

Comment:
I don't think this `clear` is needed since the `before_each` test function already calls it.